Examples/Copy-NsxIpset.ps1

```diff
@@ -0,0 +1,57 @@
+function Copy-NsxIpSet {
+    <#
+    .SYNOPSIS
+    Copies NSX IP sets from Primary NSX manager to Secondary NSX Manager
+
+    .DESCRIPTION
+    This Function Helps you to copy NSX Ip sets from One NSX Manager to Another 
+
+    .EXAMPLE
+    Copy-NsxIpSet -PrimaryNsxManager Nsx01.xyz.com -SecondaryNsxmanager Nsx02.xyz.com 
+
+    
+
+    #>
+    [cmdletBinding()] 
+    param (
+      [parameter(Mandatory = $true)]
+      [string] $PrimaryNsxManager,
+      [parameter(Mandatory = $true)]
+      [string] $SecondaryNsxManager,
+      [parameter (Mandatory = $true)]
+      [pscredential] $credential
```

Comment:
how does it handle different credentials for the primary and secondary managers?